tests/components/homematicip_cloud/test_init.py

```diff
@@ -97,15 +97,17 @@
         instance.home.modelType = "mock-type"
         instance.home.name = "mock-name"
         instance.home.currentAPVersion = "mock-ap-version"
-        instance.async_reset.return_value = mock_coro(True)
+        instance.async_reset.return_value = CoroutineMock(True)
```

Comment:
```py
instance.async_reset = CoroutineMock(return_value=True)
```

Reply:
Please clean this up on line 95 too. Then we don't need `mock_coro` anymore.

Reply:
Oh, i missed that